buildslave.rst

```diff
@@ -167,19 +175,19 @@ For OSX:
 
 For Windows:
 
-    * Add a Scheduled Task to run ``buildslave start buildarea`` as the
+   *  Add a Scheduled Task to run ``buildslave start buildarea`` as the
       buildbot user "when the computer starts up".  It is best to provide
       absolute paths to the ``buildslave`` command and the :file:`buildarea`
       directory.  It is also recommended to set the task to run in the
       directory that contains the :file:`buildarea` directory.
 
-    * Alternatively (note: don't do both!), set up the buildslave
+   *  Alternatively (note: don't do both!), set up the buildslave
```

Comment:
This change seems unrelated, but if you are going to fix the whole paragraph should be dedented 1 space, not just the ``*``.

Reply:
Except for code, indenting is supposed to be on a multiple of 3 according to https://devguide.python.org/documenting/#use-of-whitespace . The text of this list item is indented at 6, so I'm just moving the item marker from 4 to 3 as part of aligning the *enclosing* margin to column 3. You may notice that in a lot of console code I inserted "$" in a carefully-chosen column not "$ ".

So that's consistent, but now I look again, this enclosing indent looks spurious and everything should be 3 columns left of where I left it. Separate PR?